### ISSUE TITLE
Keyboard friendliness 

### ISSUE THREAD

**Issue by haji-ali** [rugk/awesome-emoji-picker#41]
When I press "Ctrl+Shift+period", the emoji picker is opened but I still have to use the mouse to focus the search box to type the keyword search. Then I have to use the mouse again to select the particular emoji. 
Is it possible to auto-focus the search box when pressing the shortcut key, then using some controls like the arrow keys (or Ctrl+arrow keys) to select the particular emoji to copy?

**Comment by rugk** [comment]
> I still have to use the mouse to focus the search box to type the keyword search

This is tracked in https://github.com/rugk/awesome-emoji-picker/issues/28.

> using some controls like the arrow keys (or Ctrl+arrow keys) to select the particular emoji to copy

Lets track this so, in this issue. And I would also really like it.

Thing is, I do not implement it myself, and use emoji-mart as a lib for the emoji picker. 
So can you please first suggest this "keyboard navigation" feature on https://github.com/missive/emoji-mart?

That said, BTW; currently, you should be able to use <kbd>Tab</kbd> to cycle through all things, but it may indeed not really be intuitive.

**Comment by rugk** [comment]
Opened an issue https://github.com/missive/emoji-mart/issues/348

Marking this as blocked by that issue (though I may look into providing/implementing at least some features here, or even upstream, by myself).

**Comment by rugk** [comment]
BTW if you are following this issue, there is a good chance this get's into our upstream library and thus later in this add-on. See/comment on https://github.com/missive/emoji-mart/pull/431 to give feedback now, if you want.

**Comment by ddidier** [comment]
Hi, this doesn't appear to be solved, is there any reason why?

**Comment by rugk** [comment]
Yes, this may be solved by https://github.com/rugk/awesome-emoji-picker/pull/128 aka an upgrade of the lib, but nobody completed it until now and I also did not have the time to look into it.
